### DISCUSSION TITLE
Great job !

### DISCUSSION THREAD

**Discussion by stefaanv** [wlcrs/huawei_solar#525] [General]
Hi wlcrs, 
You've done a great job with this integration.
I'm using it for load shifting, works like a charm.

Thanks !
Stefaan

**Comment by Roving-Ronin** [comment]
@stefaanv are you using any particular software / integration for the load shifting ? Or are you talking manual turning things on/off based upon the read outs from wlcrs ?

**Comment by stefaanv** [comment]
I'm writing my own software
Untill now that means setting charge times manually every evening for the next day.
The goal is to automate with LP calculation.
Work in progress 🤪
